content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst

```diff
@@ -1,65 +1,230 @@
+:menuselection:`Sales --> Create` to create a new quotation, and fill out the information. Add a
+:guilabel:`Customer`, click :guilabel:`Add a product` to add products to the :guilabel:`Product`
+lines, then set a :guilabel:`Quantity`.
+
+Then, click the :guilabel:`Other Info` tab. Under the :guilabel:`Delivery` section, change the
+:guilabel:`Delivery Date` to a date after the expected date, and click the
+:guilabel:`green checkmark` to confirm the date. Finally, :guilabel:`Confirm` the sales order.
+
+Click the :guilabel:`Delivery` smart button at the top of the sales order to be taken to the
+:guilabel:`Warehouse Receipt Form`. Click :guilabel:`Validate` and :guilabel:`Apply` to process all
+:guilabel:`Done` quantities and deliver the products. If the products are delivered before
+the :guilabel:`Alert Date` set on the product form, then no alerts will be created.
+
+View expiration dates for lots & serial numbers
+===============================================
+
+To view and group all products with expiration dates by lot number, go to :menuselection:`Inventory
+--> Products --> Lots/Serial Numbers`. Remove any :guilabel:`Filters` in the
+:guilabel:`Search bar`, then click :guilabel:`Group By`, :guilabel:`Add Custom Group`, and click
+the :guilabel:`drop-down` to select :guilabel:`Expiration Date.` :guilabel:`Apply` the filter. This
+will break down all perishable products, their :guilabel:`Expiration Dates`, and the assigned lot
+number.
+
+.. image:: expiration_dates/expiration-dates-group-by-dates.png
    :align: center
+   :alt: Group by expiration dates on lots and serial numbers page.
+
+To see any expiration alerts for products that are either expired (or will expire soon), remove all
+:guilabel:`Filters` from the :guilabel:`Search bar`. Then, click :guilabel:`Filters`, then
+:guilabel:`Expiration Alerts`.
+
+Expiration alerts
+-----------------
 
-Expiration Alerts
-=================
+To see expiration alerts, go to :menuselection:`Inventory --> Products --> Lots/Serial Numbers`.
+Then, click into a :guilabel:`Lot/Serial Number` containing perishable products. Once on the serial
+number form, click the :guilabel:`Dates` tab to see all expiration information about the products.
 
-You can access expiration alerts from the *inventory* app. To do so, go to :menuselection:`Master
-Data --> Lots/Serial Numbers`. There, you can use the pre-existing filter to show all the
-lots/serial numbers that have exceeded their respective alert dates.
+Click :guilabel:`Edit` to edit the form, then change the :guilabel:`Expiration Date` to today's
+date (or earlier), and :guilabel:`Save` changes. The lot number now displays a red
+:guilabel:`Expiration Alert` below it at the top of the lot form to indicate that the products in
+this lot are either expired or expiring soon. Click back to the :guilabel:`Lots/Serial Numbers`
+page (via the breadcrumbs) and click :guilabel:`Filters`, then click :guilabel:`Expiration Alerts`
+to see the new alert.
```

Comment:
To edit the form, click :guilabel:`Edit` in the upper-left corner of the form, then change the :guilabel:`Expiration Date` to the new desired date, and click :guilabel:`Save` to save those changes. 

After saving, the lot number form displays a red :guilabel:`Expiration Alert` at the top of the form to indicate that the products in this lot are either expired or expiring soon. 

From here, click back to the :guilabel:`Lots/Serial Numbers` page (via the breadcrumbs), and click :guilabel:`Filters`, then select :guilabel:`Expiration Alerts` to see the new alert.